pagopa/io-app
# chore: [IOBP-1044,IOBP-1054] Add missing outcome screen into payment onboarding

## Short description
This PR adds two missing outcome screens into payment onboarding flow.

## List of changes proposed in this pull request
- Mapped outcome `PSP_ERROR_ONBOARDING` and `BE_KO` inside the payment method onboarding flow;
- Handled the onboarding outcomes inside the `usePaymentFailureSupportModal` hook;
- Refactored the `trackAddOnboardingPaymentMethod` function to keep track of the analytics onboarding outcomes;

## How to test
- Checkout this dev-server PR: https://github.com/pagopa/io-dev-api-server/pull/444;
- Start the payment onboarding flow from the wallet screen;
- When you reach the in-app browser with the dropdown selection, choose `PSP_ERROR_ONBOARDING` or `BE_KO` and check that you see the correct screen;

## Preview
|PSP_ERROR_ONBOARDING|BE_KO|
|-|-|
|<img src="https://github.com/user-attachments/assets/b148cc63-cdd0-46dd-a7eb-3891c69c940e" width="350"/>|<img src="https://github.com/user-attachments/assets/bbaa3eea-76f2-4a21-9f93-e7d4414f9538" width="350"/>|
